scripts/dream.py

```diff
@@ -167,6 +177,23 @@ def main_loop(t2i, outdir, parser, infile):
             if not os.path.exists(opt.outdir):
                 os.makedirs(opt.outdir)
             current_outdir = opt.outdir
+        elif prompt_as_dir:
+            if prompt_as_dir == 'simple':
+                # this line will sanitize the entire prompt as one folder
+                subdir = path_filter.sub('_', opt.prompt)[:name_max].rstrip(' .')
+            else:
+                # this line will allow the prompt to create subdirectories
```

Comment:
I don't think we should allow the prompt to create subdirectories. If I ask for `a human/dragon hybrid` or something, I would not expect to get a directory named "a human" containing a directory named "dragon hybrid".

Reply:
Somewhat agree, I don't personally like the strip/preserve folders choice cause it's a little clunky and could lead to someone accidentally dropping the first word of their prompt as mentioned before. Wondering what others think about this.